lib/dest/writeContents/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var fs = require('graceful-fs');
+
+function utimes(writePath, stat, cb) {
+  if (stat.mtime) {
+    stat.atime = stat.atime || new Date();
```

Comment:
this is mutating the `atime` on the file object.  I don't think we want that, correct?